models/public/deeplabv3/deeplabv3.md

```diff
@@ -0,0 +1,71 @@
+# deeplab-v3
+
+## Use Case and High-Level Description
+
+DeepLab is a state-of-art deep learning model for semantic image segmentation. For details see [paper](https://arxiv.org/pdf/1706.05587.pdf)
+
+## Example
+
+## Specification
+
+| Metric            | Value                |
+|-------------------|----------------------|
+| Type              | Semantic segmentation|
+| GFLOPs            | 11.469               |
+| MParams           | 23.819               |
+| Source framework  | Tensorflow           |
+
+## Accuracy
+
+## Performance
+
+## Input
+
+### Original model
+
+1. Name: `ImageTensor`, shape: [1x513x513x3] - An input image in the format [BxHxWxC],
```

Comment:
Shouldn't we align this format with intel models description?

Reply:
Not necessary, we took intel model description as example, not a pattern, because there are differences in documentation intel and public model.
Besides, list with single element looks incomplete.